mirascope/cli/utils.py

```diff
@@ -15,6 +15,93 @@
 ignore_variables = {"prev_revision_id", "revision_id"}
 
 
+class PromptAnalyzer(ast.NodeVisitor):
```

Comment:
should we expand on the docstrings here?

Reply:
i added an example